app/views/service_calendars/master_calendar/pppv/_line_item_visit_input.html.haml

```diff
@@ -1,47 +0,0 @@
--# Copyright © 2011 MUSC Foundation for Research Development
```

Comment:
Broken up into 3 different files with the tab in the file name. `_template_visit_input.html.haml`, `_billing_strategy_visit_input.html.haml`, and `_calendar_visit_input.html.haml`